docs/table_design/Data_distribution.md

```diff
@@ -1,107 +1,209 @@
+> **NOTE**
+>
+> In addition to the general distribution methods, StarRocks also supports Random distribution to simplify bucketing configuration.
+
+Also, StarRocks distributes data by implementing the two level partitioning + bucketing method.
+
+- The first level is partitioning: Data within a table can be partitioned. The partitioning method can be range partitioning or list partitioning. Or you can choose not to use partitioning (the entire table is regarded as one partition).
+- The second level is bucketing: Data in a partition needs to be further distributed into smaller bucktes. The bucketing method can be hash or random bucketing.
+
+| **Distribution method**   | **Partitioning and bucketing method**                        | **Description**                                              |
+| ------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Random distribution       | Random bucketing                                             | The entire table is considered a partition. Data in the table is randomly distributed into different buckets. This is the default data distribution method. |
+| Hash distribution         | Hash bucketing                                               | The entire table is considered a partition. The data in the table is distributed to the corresponding bucket, which is based on the hash value of the data's bucketing key by using a hashing function. |
+| Range+Random distribution | <ol><li>Expression partitioning or range partitioning </li><li>Random bucketing </li></ol> | <ol><li>The data of the table is partitioned based on the ranges of values in the partitioning columns. </li><li>The data in a partition is randomly distributed across different buckets. </li></ol> |
+| Range+Hash distribution   | <ol><li>Expression partitioning or range partitioning</li><li>Hash bucketing </li></ol> | <ol><li>The data of the table is partitioned based on the ranges of values in the partitioning column.</li><li>The data in a partition is distributed to the corresponding bucket, which is based on the hash value of the data's bucketing key by using a hashing function.</li></ol> |
+| List+Hash distribution    | <ol><li>Expression partitioning or List partitioning</li><li>Hash bucketing </li></ol> | <ol><li>The data in the table is partitioned based on the values in the partitioning column.</li><li>The data in a partition is distributed to the corresponding bucket, which is based on the hash value of the data's bucketing key by using a hashing function.</li></ol> |
+| List+Random distribution  | <ol><li>Expression partitioning or List partitioning</li><li>Random bucketing </li></ol> | <ol><li>The data in the table is partitioned based on the values in the partitioning column.</li><li>The data in a partition is randomly distributed across different buckets.</li></ol> |
+
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/28993d2c-c191-4d3f-900d-0556331fbb19)
See the Chinese version. It seems the English version differs from the Chinese version.